generator-integration-test/src/test/kotlin/com/lsdconsulting/stub/integration/get/GetRestControllerStandardResponseIT.kt

```diff
@@ -405,6 +434,8 @@ class GetRestControllerStandardResponseIT : BaseRestControllerIT() {
         assertThat(response.body?.name, `is`(name))
         underTest.verifyResourceWithOptionalBooleanRequestParam(1, false)
         underTest.verifyResourceWithOptionalBooleanRequestParam(false)
+        underTest.verifyResourceWithOptionalBooleanRequestParamNoInteraction(true)
+//        underTest.verifyResourceWithOptionalBooleanRequestParamNoInteraction(null) // Doesn't pass as this effectively verifies the resource was called with no parameter which is not true
```

Comment:
Maybe remove this or `assertThrows`?

Reply:
Good suggestion. Added an assert. Thanks